apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -355,18 +355,28 @@ export class PackageJsonUpdater {
       }
       console.log(`Querying NPM registry for latest version of "${packageName}"...`);
```

Comment:
This log message is no longer accurate. This should go inside the condition after the loop.

Same with the code [here](https://github.com/microsoft/rushstack/pull/1612/files#diff-51550989f206f917116c6152dd70b1e9R366-R371)